Formula/ghc.rb

```diff
@@ -86,6 +86,8 @@ def install
 
     # MPN_PATH: The lowest common denominator asm paths that work on Darwin,
     # corresponding to Yonah and Merom. Obviates --disable-assembly.
+    # TODO: adjust this based on the fact Mojave only supports Nehalem and
+    # newer processors.
```

Comment:
I'd suggest we align this on the `gmp` formula itself: pass `--build=core2-apple-darwin#{`uname -r`.to_i}` (and possibly `nehalem` on Mojave) to the `configure` script. (This in turn sets the `MPN_PATH` to what GMP itself thinks is more suitable for a given processor.)